README.md

```diff
@@ -1 +1,260 @@
-# myOwnAwesomeBooks
\ No newline at end of file
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="/banner.jpg" alt="logo" />
+  <br/>
+
+  <h1><b>Awesome-books</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome-books] <a name="about-project"></a>
+
+<!-- > Describe your project in 1 or 2 sentences. -->
+
+**[Awesome-books]** is a project that makes a list of your favorite books.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- > Describe the tech stack and include only the relevant sections that apply to your project. -->
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Linters</li>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Javascript</li>
+    <li>Github pages</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>No server</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>No database</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- **[Use medium-fidelity warframe to build the project]**
+- **[Responsive design for multiple screens]**
+- **[Use of javascript to change the content of the page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- > Add a link to your deployed project. -->
+
+- [Awesome Books!]()
+
```

Comment:
- [ ] @fdezcaminero great work on your documentation, however, you added a live demo that leads to the repo instead of a live demo, Kindly consider making the link work or you have an option of removing this section since it's not a project requirement at this milestone.

Reply:
This change has been done well